src/utils/index.ts

```diff
@@ -1 +1,38 @@
+import { useEffect, useRef, useState } from "react";
 export * from "./css";
+
+const obtainVarName = (varNotation: string) => {
+  if (!varNotation.includes("var(")) {
+    return varNotation;
+  }
+
+  const matches = varNotation.match(/\(([^)]+)\)/);
+
+  if (matches && matches?.length > 1) {
```

Comment:
in the if we have matches[1], TS will yell at you that you accessing something that can be undefined which is true because you can pass condition `matches.length > 0` and fail at `matches[1]`